Sm0k3s/SendIT
# #162037844 Builds endpoint for creating a parcel

#### What does this PR do?
Finishes the endpoint to post a parcel delivery order on the app
#### Description of Task to be completed?
Finish endpoint to post parcels
```POST /api/v2/parcels ```
#### How should this be manually tested?
Clone this repo and create a python3 virtualenv 
```bash
$ python3 -m venv venv
$ source venv/bin/activate
```
After activating the environment, install the dependencies 
```
$ pip install -r requirements.txt
```

Open postman and test the endpoint 

```bash
POST /api/v2/parcels 

The expect payloads is 
       {
			"title":"kuku",
			"destination": "cuckoo land",
			"weight": 2,
			"pickup_location": "Nairobi",
			"description":"oooh yum yum"
		}
```
#### What are the relevant pivotal tracker stories?
[#162037844](https://www.pivotaltracker.com/n/projects/2213547/stories/162037844)
